wordpress__api/Posts.d.ts

```diff
@@ -120,6 +120,15 @@ declare module '@wordpress/api/posts' {
 		tags?: number[];
 	}
 
+	/**
+	 * Update a Post.
+	 *
+	 * @link https://developer.wordpress.org/rest-api/reference/posts/#arguments-4
```

Comment:
Let's point to the has of the section instead of the table in case the `id` of the table changes.

```suggestion
	 * @link https://developer.wordpress.org/rest-api/reference/posts/#update-a-post
```